src/index.ts

```diff
@@ -12,9 +12,11 @@ export {AppBridgeContext} from './utilities/app-bridge';
 
 /* eslint-disable @typescript-eslint/camelcase */
 export {
+  buildCustomProperties as UNSTABLE_buildCustomProperties,
   buildColors as UNSTABLE_buildColors,
   roleVariants as UNSTABLE_roleVariants,
   toCssCustomPropertySyntax as UNSTABLE_toCssCustomPropertySyntax,
   Tokens as UNSTABLE_Tokens,
+  inverse,
```

Comment:
Not a fan of exporting this. As Tim mentioned, being able to specify `mode: 'inverse'` so this flipping can be  encapsulated within the ThemeProvider would be preferable to putting that responsibility on the user.

Reply:
Agreed